Cargo.toml

```diff
@@ -1,13 +1,18 @@
 [workspace]
 members = [
-  "crates/*",
-  "bindings/prql-elixir/native/prql",
-  "bindings/prql-java",
-  "bindings/prql-js",
-  "bindings/prql-lib",
-  "bindings/prql-python",
+  "prqlc/crates/ast",
+  "prqlc/crates/compiler",
+  "prqlc/crates/compiler-macros",
+  "prqlc/crates/parser",
```

Comment:
I would note from `@matklad` in https://matklad.github.io/2021/08/22/large-rust-workspaces.html

> Don’t succumb to the temptation to strip common prefix from folder names. If each crate is named exactly as the folder it lives in, navigation and renames become easier. Cargo.tomls of reverse dependencies mention both the folder and the crate name, it’s useful when they are exactly the same.